docs/application/dotnet/guides/overview.md

```diff
@@ -1,52 +1,16 @@
-
-- `AppLinkEntry`
-- `OpenGLView`
-
-A list of limitations is available at [Current Xamarin.Forms limitations](xamarin-forms-limitations.md).
+Tizen .NET provides a rich set of interfaces allowing you to build compelling TV, mobile, and wearable (preview) applications which achieve native performance.
+Tizen.Wearable.CircularUI supports Tizen wearable-specific user interfaces and TizenFX API support Tizen platform-specific functionalities.
 
 ## Tizen.Wearable.CircularUI
 
-[Tizen.Wearable.CircularUI](https://samsung.github.io/Tizen.CircularUI/index.html) provides Tizen wearable-specific user interfaces. It is a set of extension APIs of Xamarin.Forms. Tizen.Wearable.CircularUI APIs are supported only on Tizen wearable devices with Tizen .NET support, unlike the Xamarin.Forms which support cross platforms.
+Tizen.Wearable.CircularUI is a set of extension APIs of Xamarin.Forms. Tizen.Wearable.CircularUI APIs are supported only on Tizen wearable devices with Tizen .NET support, unlike the Xamarin.Forms which support cross platforms.
 
-Using these APIs, you can easily and efficiently add Tizen wearable-specific user interfaces to your Tizen wearable application. For more information about these APIs, see [Tizen.Wearable.CircularUI GitHub](https://github.com/Samsung/Tizen.CircularUI).
+You can easily create your Tizen wearable applications with [Tizen.Wearable.CircularUI guides](https://samsung.github.io/Tizen.CircularUI/guide/Overview.html). For more information about these APIs, see [Tizen.Wearable.CircularUI GitHub](https://github.com/Samsung/Tizen.CircularUI).
 
 ## TizenFX API
 
-The TizenFX API allows applications to call into platform-specific functionality from the shared code. This functionality enables Xamarin.Forms applications to do things a native application can do, without causing the portable part of the application to become littered with operating system-specific details.
-
-The following table specifies the API Level supported by each version of the Tizen platform:
-
-| Platform Version | API Level | Notes                                                        |
-| ---------------- | --------- | ------------------------------------------------------------ |
-| Tizen 4.0 M2     | 4         | [TizenFX API Reference](https://samsung.github.io/TizenFX/API4/) |
-| Tizen 5.0 M2     | 5         | [TizenFX API Reference](https://samsung.github.io/TizenFX/API5/) |
-
-
-### TizenFX API Guides
-
 TizenFX API introduce various features for Tizen .NET applications. The TizenFX API allows applications to call into platform-specific functionality from the shared code. This functionality enables Xamarin.Forms applications to do things a native application can do, without causing the portable part of the application to become littered with operating system-specific details.
 
 You can use the following TizenFX features in your .NET applications:
```

Comment:
Suggestion, or Cross-check : Below features have only one subject under that category now. As like the case of "Localization", how about directly linking to the sub page, instead of linking to the overview.md?
 - Data Storage and Management
 - Natural Language Processing

Reply:
When NLP is added, we should've check menu and overview page.
Anway, I will update related files.
Thanks.